e2e/cypress/integration/messaging/mobile_profile_popover_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Profile popover', () => {
+    before(() => {
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+
+        // # Post a new message to ensure there will be a post to click on
+        cy.postMessage('Test message');
+    });
+
+    afterEach(() => {
+        // Set viewport back to desktop view to make sure changeMessageDisplaySetting works between tests
+        cy.viewport('macbook-13');
```

Comment:
If you use the API command (which you should) to change the message display preference, this afterEach can be removed.

Reply:
Yup, you're right, will be sure to remove that as I use the API commands instead of the UI ones